tests/python/sgtk_integration_test.py

```diff
@@ -144,21 +147,67 @@ def create_or_find_project(cls, name, entity=None):
         :param dict entity: Entity dictionary for the project if it needs to be created.
 
         .. note:
-            The actual name of the project might be different than the name passed in if you
+            The actual name of the project will be different than the name passed in. As such,
+            always use the name returned from the entity.
+
+        :returns: Entity dictionary of the project.
+        """
+        return cls.create_or_find_entity("Project", "Testing: %s" % name, entity)
```

Comment:
Prefixing Project that are for testing with Testing: makes the project list a bit more readable.